client/src/app.tsx

```diff
@@ -336,53 +336,53 @@ export const PinBoardApp = ({ apolloClient, userEmail }: PinBoardAppProps) => {
   return (
     <TelemetryContext.Provider value={sendTelemetryEvent}>
       <ApolloProvider client={apolloClient}>
-        <Global styles={agateFontFaceIfApplicable} />
-        <HiddenIFrameForServiceWorker iFrameRef={serviceWorkerIFrameRef} />
-        <root.div
-          onDragOver={(event) =>
-            isGridDragEvent(event) && event.preventDefault()
+        <GlobalStateProvider
```

Comment:
changes in this file seem a lot but its just moving the `GlobalStateProvider` slightly further up the hierarchy so the `HiddenIFrameForServiceWorker` has access to the global state (see that changes in that file).